pages/studentAssessments/studentAssessments.sql

```diff
@@ -155,5 +155,4 @@ WHERE
 ORDER BY 
     CASE WHEN $assessments_group_by = 'Module' THEN assessment_module_number END, 
     CASE WHEN $assessments_group_by = 'Module' THEN assessment_module_id END,
-    assessment_set_number, assessment_order_by, assessment_id, assessment_instance_number 
-    NULLS FIRST;
+    assessment_set_number, assessment_order_by, assessment_id, assessment_instance_number;
```

Comment:
@mwest1066 I grepped for `NULLS FIRST` and this was the only other occurrence. I believe it's safe to remove, but would appreciate a second look!

Reply:
I agree this is also safe to remove. (I hope I don't regret saying this 😄)